rails/docker-compose.rails.yml

```diff
@@ -52,30 +11,42 @@ services:
             ACRA_MASTER_KEY: ${ACRA_SERVER_MASTER_KEY:-N0MyZEVCRGY1cHA4ODRSVHp3c3BNeGtDekxaV3BhTHI=}
         volumes:
             - ./.acrakeys:/keys
+            - ../_common/ssl/acra-client/:/ssl:ro
         command: >-
-            --client_id=${ACRA_CLIENT_ID:-testclientid}
+            --client_id=""
+            --tls_cert=/ssl/acra-client.crt
             --generate_acrawriter_keys
             --keys_output_dir=/keys/acra-server
             --keys_public_output_dir=/keys/acra-writer
             --keystore=v1
 
     # Postgresql container
     postgresql:
-        image: postgres:13
+        # Build and run the container based on the official postgresql image
+        # with an additional configuration in accordance with djangoproject
+        # requirements
+        build:
+            context: ../
+            dockerfile: _common/pgsql-django/pgsql-django.dockerfile
+            args:
+                VCS_URL: ${COSSACKLABS_ACRAENGDEMO_VCS_URL}
+                VCS_REF: ${COSSACKLABS_ACRAENGDEMO_VCS_REF}
+                VCS_BRANCH: ${COSSACKLABS_ACRAENGDEMO_VCS_BRANCH}
+                BUILD_DATE: ${COSSACKLABS_ACRAENGDEMO_BUILD_DATE}
         # INSECURE!!! You MUST define your own DB name and credentials
         environment:
             # Admin user
-            POSTGRES_USER: ${POSTGRES_USER:-rubygems}
-            POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-rubygems}
-            POSTGRES_DB: ${POSTGRES_DB:-rubygems}
+            POSTGRES_USER: ${POSTGRES_USER:-postgres}
+            POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-test}
+            POSTGRES_DB: ${POSTGRES_DB:-test}
+            POSTGRES_DJANGO_PASSWORD: ${POSTGRES_DJANGO_PASSWORD:-test}
```

Comment:
For now, we used [postgresql-image](https://github.com/cossacklabs/acra-engineering-demo/blob/master/_common/pgsql-django/pgsql-django-configure.sh#L32) from _common which is used for django demos firstly and `POSTGRES_DJANGO_PASSWORD` is required to provide here. What do you think to do, create another image or try to make this one general

Reply:
imho, better to use separate dockerfile. as I understand, this image placed in `_common/pgsql-django` because of we have several examples with django. this one is not django based and may be used by users who don't know django. so finally, I think better to use separate dockerfile placed in `rails` folder.
@shadinua , what do you think?

Reply:
The same situation now with python demo, we used the same `_common/pgsql-django` image as well.

Reply:
I see a couple different approaches:
1. To build a separate docker image. As for me, this is not the most efficient way due to it leads to support efforts for 2×, 3×, ... mostly same images.
2. I'm looking into the [pgsql-django-configure.sh](https://github.com/cossacklabs/acra-engineering-demo/blob/master/_common/pgsql-django/pgsql-django-configure.sh#L32) and on my mind this image, actually, is pretty common to be named `pgsql-django`. As far as we tend to re-use it, I'd suggest to:
  - rename the image to `pgsql-playground`
  - rename the file `pgsql-django-configure.sh` to `pgsql-configure.sh`
  - make a little changes to make specific configurations only if appropriate env vars are set; for example, do django preparations only if `POSTGRES_DJANGO_PASSWORD` was set
3. Create separate entrypoint files for each kind of usage.

IMHO, the better way is 2, 3 also is good enough.

Reply:
Thanks @shadinua, I lean towards the second option too. @Lagovas

Reply:
I agree. 2 and 3 options looks good. Third option better if we have a lot of specific logic and we can place some common entry.sh for next re-usage and demo specific entry.sh. But lets start with second way.